modules/shared/nRF52840/part1.ld

```diff
@@ -232,9 +232,15 @@ SECTIONS
         _sbss = ALIGN(., 4);
         link_bss_location = ALIGN(., 4);
         __bss_start__ = _sbss;
+        /* Store the BSS sections for the RTOS first */
+        __rtos_capture_start = .;
+         *tasks.o(.bss COMMON .bss*)
+         *timers*.o(.bss COMMON .bss*)
```

Comment:
I’m not sure what this ld file change has to do with the other system_version changes. Suggest splitting into a separate PR.

Reply:
This is for memfault FreeRTOS introspection support, all task lists etc which are static variables are collected in a single coalescent location.